perun-core/src/main/java/cz/metacentrum/perun/core/api/AuthzResolver.java

```diff
@@ -1289,7 +1289,7 @@ public static void unsetRole(PerunSession sess, User user, PerunBean complementa
 
 		try {
 			if(!authorizedToManageRole(sess, complementaryObject, role)) {
-				throw new PrivilegeException("You are not privileged to use the method unsetRole.");
+				throw new PrivilegeException("You are not privileged to use the method unsetRole. "+authorizedToManageRole(sess, complementaryObject, role));
```

Comment:
Why is this here

Reply:
Good catch, just a leftover from debugging